playbooks/upload_to_swift.yml

```diff
@@ -44,79 +49,81 @@
       tags:
         - skip_ansible_lint
 
-    - name: Create a public Cloud Files container
+    - name: Create a cloudfiles container
```

Comment:
Total nit, but we still have references to `Cloud Files` and `CloudFiles` below. I'd probably just standardise on `Cloud Files` since I think that's what the product is called.

Reply:
Done. That was kinda bothering my OCD too.